Makefile

```diff
@@ -1190,3 +1189,10 @@ shp/ks/congress-ungrouped.shp: shp/us/congress-ungrouped.shp
 	mkdir -p $(dir $@)
 	ogr2ogr -f 'ESRI Shapefile' -where "STATEFP = '20'" $(dir $@) $<
 
+#
+# Historical JSON
+#
+data/simplified-historical.json: data/historical.json
+	cat data/historical.json \
+		| ./simplify-historical \
+		> $@
```

Comment:
Simplifying the candidate structure seems like low hanging fruit here. Gets the before-gzip size to 1.7mb, after gzip to 160k.